code-challenges.test.js

```diff
@@ -15,42 +17,147 @@
+// }
+
+
+//* RAN TEST (RESULTS BELOW): *//
+
+
+// FAIL  ./code-challenges.test.js
+// leastThenGreater
+//   ✕ takes in an array and returns a new array of only odd numbers sorted from least to greatest (2 ms)
+
+// ● leastThenGreater › takes in an array and returns a new array of only odd numbers sorted from least to greatest
+
+//   expect(received).toEqual(expected) // deep equality
+
+//   Expected: [-9, 7, 9, 199]
+//   Received: "9,7,true,hey,7,199,-9,hola-9,0,false,true,4,7,7,8,9,hey,199,holatrue,false,false,true,false,true,true,false,true,true,true,true"
+
+// ** NOTES TO INSTRUCTOR **: I'm not sure what the Jest test result was saying I did incorrectly, I found the response odd/confusing. Any clarity on this matter? 
 
 // --------------------3) Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
 
 // a) Create a test with expect statements for each of the variables provided.
 
-const numbersToAdd1 = [2, 4, 45, 9]
-// Excpected output: [2, 6, 51, 60]
+describe("onlySum", () => {
+    it("takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.", () => {
+        const numbersToAdd1 = [2, 4, 45, 9]
+        // Excpected output: [2, 6, 51, 60]
+        const numbersToAdd2 = [0, 7, -8, 12]
+        // Expected output: [0, 7, -1, 11]
+        const numbersToAdd3 = []
+        // Expected output: []
+        expect(onlySum([2, 4, 45, 9])).toEqual([2, 6, 51, 60])
+        expect(onlySum([0, 7, -8, 12])).toEqual([0, 7, -1, 11])
+        expect(onlySum([])).toEqual([])
+    })
+})
 
-const numbersToAdd2 = [0, 7, -8, 12]
-// Expected output: [0, 7, -1, 11]
-
-const numbersToAdd3 = []
-// Expected output: []
+//* RAN TEST (RESULTS BELOW): *//
 
+// ReferenceError: onlySum is not defined
 
 // b) Create the function that makes the test pass.
+
+// Create a function that takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+
+//Pseudocode:
+// Declare a function called onlySum. 
+// Set the parameter as an array.
+// Use the .forEach method accumlate sum of array elements. 
+// Use the .map to return expectd results.
+
+const onlySum = (sumArray) => {
+    let sum = 0;
+    return sumArray.forEach(value => 
+        sum+= value) + sumArray.map 
+    
+}
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// FAIL  
+// onlySum
+//     ✕ takes in an array and returns an array of the accumulating sum. An empty array should return an empty array. (1 ms)
+// ● onlySum › takes in an array and returns an array of the accumulating sum. An empty array should return an empty array.
+
+// expect(received).toEqual(expected) // deep equality
+
+// Expected: [2, 6, 51, 60]
+// Received: "undefinedfunction map() { [native code] }"
+
+// ** NOTES TO INSTRUCTOR **: Get a similiar issue to the one before, I'm guessing my logic is a bit off somehow? 
```

Comment:
This one is  harder, but you were so close!  Remember that forEach() does not return anything. Just move up that .map() and you'd have had it.  Great thought process and very, very close!